subsys/mgmt/ec_host_cmd/backends/ec_host_cmd_backend_spi_stm32.c

```diff
@@ -260,6 +266,34 @@ static int expected_size(const struct ec_host_cmd_request_header *header)
 	return sizeof(*header) + header->data_len;
 }
 
+#ifdef CONFIG_PM
+static void ec_host_cmd_pm_policy_state_lock_get(struct ec_host_cmd_spi_ctx *hc_spi)
+{
+	if (!hc_spi->pm_policy_lock_on) {
+		hc_spi->pm_policy_lock_on = true;
```

Comment:
>There can only be one host command in flight at a time, correct?

True. I'm just afraid of missing/not handling in time one interrupt for any reason. That would mean the system won't ever suspend because the PM counter is increased to 2.